sig/lrama/states/item.rbs

```diff
@@ -0,0 +1,44 @@
+module Lrama
+  class States
+    class Item
```

Comment:
It is not subject to Item class type checking. I'll define the type in another PR.

Reply:
imo) If it's TODO, I think it's better to add a TODO comment because if for some reason you can't send a follow up PR, someone can notice this later. But, it is not a strong opinion.

Reply:
Well, sure. Thanks for the opinion.
However, I can get a PR out immediately when this case. I will write a TODO next time.